src/hotspot/os/linux/os_linux.cpp

```diff
@@ -401,7 +401,7 @@ bool os::Linux::get_tick_information(CPUPerfTicks* pticks, int which_logical_cpu
 // Returns the kernel thread id of the currently running thread. Kernel
 // thread id is used to access /proc.
 pid_t os::Linux::gettid() {
-  int rslt = syscall(SYS_gettid);
+  int64_t rslt = syscall(SYS_gettid);
```

Comment:
```suggestion
  long rslt = syscall(SYS_gettid);
```

Reply:
I am in favour of `long` too, since that is what syscall returns. int64_t does not seem to have any obvious relation to the call. And there is no reason to need 64-bit on 32-bit Linux.

Reply:
I suppose we can use long in platform code.  I'll change it to long.